examples/cpp/csm/observability/README.md

```diff
@@ -0,0 +1,35 @@
+# gRPC C++ CSM Hello World Example
+
+This CSM example builds on the [Hello World Example](https://github.com/grpc/grpc/tree/master/examples/cpp/helloworld) and changes the gRPC client and server to accept configuration from an xDS control plane and test CSM observability.
```

Comment:
Only client takes xDS Configuration IIUC.

Reply:
good point

Reply:
It still says both :).